selfdrive/car/interfaces.py

```diff
@@ -159,6 +159,10 @@ def get_std_params(candidate, fingerprint):
     ret.longitudinalActuatorDelayLowerBound = 0.15
     ret.longitudinalActuatorDelayUpperBound = 0.15
     ret.steerLimitTimer = 1.0
+    
+    # No Torque Interceptor by default
+    ret.enableTorqueInterceptor = False
```

Comment:
put it behind toggle `dp_mazda_ti`?

Reply:
Yes, will look at this later today. Never messed with toggles, but I can figure it out by looking at your code for other toggles. 😜

Reply:
Fixed in last commit, let me know if it's not right.